### PR TITLE
V8.3

### LINKED ISSUE #54: Add GEN token
Add GEN token to all network (except GANACHE).
0x543Ff227F64Aa17eA132Bf9886cAb5DB55DCAddf

### PR DESCRIPTION
**Pull request by ben-kaufman**
Replaces #51 
Fix #54

### CODE REVIEW

File: migrate-demo-test.js [hunk 1]
```diff
@@ -98,13 +98,44 @@ async function migrateDemoTest ({ web3, spinner, confirm, opts, migrationParams,
   const DAOToken = await avatar.methods.nativeToken().call()
   const Reputation = await avatar.methods.nativeReputation().call()
 
+  const DemoDAOToken = await new this.web3.eth.Contract(
```
Comment (orenyodfat) [inline migrate-demo-test.js]: Why not just call it DAOToken ?
Why Demo ?

Comment (ben-kaufman) [reply migrate-demo-test.js]: Just to differentiate it to make sure it's not colliding (and won't in the future).

**Comment by orenyodfat** [comment]
It want ..it is under organs

**Comment by orenyodfat** [comment]
Won’t

**Comment by ben-kaufman** [comment]
The variable name is in a broader scope... organs is just an object inside the test

**Comment by ben-kaufman** [comment]
Both are good imo, @orenyodfat ?

**Comment by orenyodfat** [comment]
What ever

File: migrate-demo-test.js [hunk 2]
```diff
@@ -98,13 +98,44 @@ async function migrateDemoTest ({ web3, spinner, confirm, opts, migrationParams,
...
+
   return {
     test: {
       name: orgName,
       Avatar,
       DAOToken,
       Reputation,
-      proposalId
+      proposalId,
+      organs: {
```
Comment (orenyodfat) [inline migrate-demo-test.js]: The organs/contracts should be out of the 'test' scope.

**Pull request merged**